samples/dfu/nrf5340/Kconfig

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2022 INTELLINIUM <giuliano.franchetto@intellinium.com>
```

Comment:
@carlescufi  what should we do WRT copyrights?

Reply:
This must be changed @GiulianoFranchetto. When you sign the CLA you effectively renounce to your own ownership of the code (that's in part why this repository is not open source in its traditional definition), so you should just remove this line and all similar ones.